workspace.json

```diff
@@ -3804,6 +3804,27 @@
             "command": "yarn ts-node -P apps/skilavottord/ws/tsconfig.app.json apps/skilavottord/ws/src/buildSchema.ts"
           }
         },
+        "migrate": {
+          "builder": "@nrwl/workspace:run-commands",
+          "options": {
+            "command": "../../../node_modules/.bin/sequelize-cli db:migrate",
```

Comment:
you should skip the `../../../node_modules/.bin` part, here and your other changes

Reply:
Has this changed. this is same as in judical-system.?

Reply:
I have done the changes.